ouroboros/__builtins__.py

```diff
@@ -135,6 +135,30 @@ def __exit__(self, type, value, tb):
 
 sentinel = object()
 
+#min
+def min(object):
+	"""
+		This method will take an iterator and return the minimum value from there.
+	"""
+	if type(object) is int:
+		raise TypeError("'int' object is not iterable")
+	elif type(object) is float:
+		raise TypeError("'int' object is not iterable")
```

Comment:
This line, should be:

`raise TypeError("'float' object is not iterable")`

Reply:
Yes my bad will fix it soon.